pepper-apis/api-build.sh

```diff
@@ -220,6 +220,64 @@ function docker_nginx_push {
     docker push "broadinstitute/pepper-nginx:$tag"
 }
 
+function print_usage() {
+    echo 'usage: api-build.sh <version> <environment> <dir> <option>...'
```

Comment:
Considering using `$0`, `$(basename $0)`, or `` `basename $0` `` here instead of hardcoding the script name.

Reply:
sure